app/src/main/res/layout/activity_home.xml

```diff
@@ -18,26 +18,20 @@
             android:text="\@username"
             android:textSize="20sp"/>
 
-        <android.support.v7.widget.CardView
+
+        <com.google.android.gms.maps.MapView
+            android:id="@+id/map"
             android:layout_width="200dp"
             android:layout_height="200dp"
-            app:cardCornerRadius="100dp"
-            android:id="@+id/map_cardView"
             android:layout_below="@+id/username"
-            android:layout_centerHorizontal="true">
-
-            <com.google.android.gms.maps.MapView
-                android:id="@+id/map"
-                android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                />
-        </android.support.v7.widget.CardView>
+            android:layout_centerHorizontal="true"
+            />
```

Comment:
#nitpicky just personally not a fan of this extra space - though I'll say, I think there's a case for it actually being easier to refactor if you leave an extra space after the last property like this